compiler_gym/service/proto/basic.proto

```diff
@@ -0,0 +1,58 @@
+// The CompilerGym service interface description.
+//
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+syntax = "proto3";
+
+package compiler_gym;
+
+option cc_enable_arenas = true;
+option go_package = "compiler_gympb";
+option java_multiple_files = true;
+option java_outer_classname = "BasicProto";
+option java_package = "com.compiler_gym";
+
+message BooleanTensor {
+  repeated int64 shape = 1;
+  // Flattened tensor of shape `shape` with C-like index order.
+  repeated bool value = 2;
+}
+
+message ByteTensor {
+  repeated int64 shape = 1;
+  // Flattened tensor of shape `shape` with C-like index order.
+  bytes value = 2;
+}
+
+message Int32Tensor {
+  repeated int64 shape = 1;
+  // Flattened tensor of shape `shape` with C-like index order.
+  repeated int32 value = 2;
+}
```

Comment:
I don't think we need both Int32 and Int64 types. They have [the same size](https://developers.google.com/protocol-buffers/docs/encoding#varints) on the wire, and Python numbers don't overflow

Reply:
I removed the int32 version.
My intention was that this information would help when translating the message to an action/observation. Then it will be clear that `numpy.ndarray` should have int64 `dtype`.